dashboards/google-kubernetes-engine/gke-workloads-at-risk.json

```diff
@@ -211,117 +223,57 @@
               }
             ],
             "timeSeriesQuery": {
-              "timeSeriesQueryLanguage": "fetch k8s_container::kubernetes.io/container/memory/request_utilization\n| ${project_id}\n| ${location}\n| ${cluster_name}\n| filter (resource.namespace_name !~ '(kube|istio|gatekeeper|gke|gmp)-system|gke-managed-filestorecsi|gke-mcs' && metric.memory_type == 'non-evictable')\n| every 5m\n| group_by\n    [resource.project_id, resource.location, resource.cluster_name, resource.namespace_name, metadata.system_labels.top_level_controller_name, metadata.system_labels.top_level_controller_type, resource.container_name],.max\n| { filter value_request_utilization_max >= 1\n| row_count\n  ; ident | row_count}\n| ratio\n| scale '%'",
+              "timeSeriesQueryLanguage": "fetch k8s_container::kubernetes.io/container/memory/request_utilization\n| ${project_id}\n| ${location}\n| ${cluster_name}\n| ${namespace_name}\n| filter (resource.namespace_name !~ '(kube|istio|gatekeeper|gke|gmp)-system|gke-managed-filestorecsi|gke-mcs' && metric.memory_type == 'non-evictable')\n| every 5m\n| group_by\n    [resource.project_id, resource.location, resource.cluster_name, resource.namespace_name, metadata.system_labels.top_level_controller_name, metadata.system_labels.top_level_controller_type, resource.container_name],.max\n| { filter value_request_utilization_max >= 1\n| row_count\n  ; ident | row_count}\n| ratio\n| scale '%'",
               "unitOverride": ""
             }
-          },
-          "title": "Memory under-provisioned"
-        },
-        "width": 2,
-        "xPos": 4,
-        "yPos": 0
+          }
+        }
       },
       {
-        "height": 4,
+        "xPos": 0,
+        "yPos": 16,
+        "width": 24,
+        "height": 16,
         "widget": {
+          "title": "Workloads without memory requested - MiB used",
           "timeSeriesTable": {
-            "columnSettings": [
-              {
-                "column": "resource.project_id",
-                "visible": true
-              },
-              {
-                "column": "resource.location",
-                "visible": true
-              },
-              {
-                "column": "resource.cluster_name",
-                "visible": true
-              },
-              {
-                "column": "resource.namespace_name",
-                "visible": true
-              },
-              {
-                "column": "metadata.system.top_level_controller_name",
-                "visible": true
-              },
-              {
-                "column": "metadata.system.top_level_controller_type",
-                "visible": true
-              },
-              {
-                "column": "value",
-                "visible": true
-              }
-            ],
```

Comment:
For some reason, dropping these column settings and relying on "default columns" seems to have collapsed this down to just"resource.cluster_name" on my end. Did you observe this too?

![image](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/assets/6540967/aa1a55b2-4e33-491f-b839-770f32a20e20)

Reply:
Odd. I guess custom MQL vs builder is handled differently.